features/admin.applications.v1/models/application.ts

```diff
@@ -931,3 +931,11 @@ export enum SubjectTypes {
     PUBLIC = "public",
     PAIRWISE = "pairwise"
 }
+/**
+ * Interface for the share application status response.
+ */
+export interface ShareApplicationStatusResponse {
```

Comment:
This is a generic interface we can use for all other async operations, isn't it? if yes, lets move this to admin.core.v1

Reply:
Bump